src/xdg.c

```diff
@@ -7,6 +7,25 @@
 #include "view.h"
 #include "workspaces.h"
 
+struct wlr_xdg_surface *
+view_get_xdg_surface(struct view *view)
```

Comment:
I like to prefix extern functions with the name of the translation unit file (e.g. in `xdg.c` extern functions start with `xdg_`).
So, either put this function in `view.c` or call is something like `xdg_surface_from_view()`. Before I got too tired last night I started with the latter approach, but really don't mind which.

Reply:
Fair enough, I'll rename the new functions to `xdg_surface_from_view()` + `xwayland_surface_from_view()` etc.